.github/workflows/issue-opened.yml

```diff
@@ -10,9 +15,42 @@ permissions:
   id-token: write
 
 jobs:
+  config:
+    runs-on: "ubuntu-latest"
+    outputs:
+      has-secrets: ${{ steps.check.outputs.has-secrets }}
+    steps:
+      - name: "Check for secrets"
+        id: check
+        shell: bash
+        run: |
+          if [ -n "${{ (secrets.GRAFANA_DELIVERY_BOT_APP_ID != '' &&
```

Comment:
why the check for those secrets is still needed?

Reply:
I'll change them to check for the repository name